drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/pom.xml

```diff
@@ -653,174 +805,6 @@
         </plugins>
       </build>
     </profile>
-
-    <!-- Profile to build GWT code and package WAR for Community builds -->
-    <profile>
-      <id>kogito</id>
-      <activation>
-        <property>
-          <name>!productized</name>
```

Comment:
ant the same here, what about using `kogitoProductized` :thinking:

Reply:
@Ginxo answered in the other thread: https://github.com/kiegroup/kie-wb-common/pull/3541#discussion_r558269910